system-test/system.ts

```diff
@@ -116,29 +99,25 @@ async function getGCFClient() {
  */
 async function pruneFunctions(sessionOnly: boolean) {
   console.log('Pruning leaked functions...');
-  const projectId = await google.auth.getProjectId();
-  const res = await gcf.projects.locations.functions.list({
+  const [fns] = await gcf.listFunctions({
```

Comment:
you know what I'm gonna suggest here right? :)

Reply:
(but in this particular case you might not need async iteration since you want them all in a list for `Promise.all` anyway)